cmd/bridge/main.go

```diff
@@ -425,10 +429,15 @@ func main() {
 				HeaderBlacklist: []string{"Cookie", "X-CSRFToken"},
 				Endpoint:        &url.URL{Scheme: "https", Host: openshiftAlertManagerHost, Path: "/api"},
 			}
+			srv.AlertManagerUserWorkloadProxyConfig = &proxy.Config{
```

Comment:
we should be also setting this server proxy config field, in case of `off-cluster` deployment
Check https://github.com/openshift/console/blob/master/cmd/bridge/main.go#L479-L492